regenerate/quantize.js

```diff
@@ -64,6 +64,13 @@ zlib.gunzip(timezoneBuffer, function(err, data) {
 
 
 function coverTile(zone, done) {
+  if (!zone.properties.tzid.toLowerCase().includes('america')) {
+    done();
+    console.info(`Ignored zone: ${zone.properties.tzid}`);
+    return;
+  }
+  console.info(`Processing zone: ${zone.properties.tzid} with ${zone.geometry.coordinates[0].length} points`)
```

Comment:
You could make a timeout for problematic zones, ie. You could place a code like this inside the "forEach" below:

```
const MAX_TIME = 30000
const id = tile.join('/')
try {
  await Promise.race([
    new Promise( rs => {
      ///
      let poly = turf.polygon(tilebelt.tileToGeoJSON(tile).coordinates)
      let overlap = turf.area(turf.intersect(zone, poly))
      if (!tiles[id] || tiles[id].overlap < overlap)
        tiles[id] = { name: zone.properties.tzid, overlap: overlap }
      ///
      rs()
    }),
    new Promise(( _, rj ) => setTimeout(() => rj( new Error( 'Exec timeout' )), MAX_TIME ))
  ])
} catch (e) {
  console.log(e, id)
}
```

Reply:
@milo-ft testing...